swagger/v1/defendant.json

```diff
@@ -95,6 +95,9 @@
         "national_insurance_number": {
           "$ref": "#/definitions/nino"
         },
+        "is_linked": {
+          "$ref": "#/definitions/is_linked"
```

Comment:
where is the `#/definitions/is_linked` defined - can't find

Reply:
https://github.com/ministryofjustice/laa-court-data-adaptor/pull/108/files#diff-b5b19d5c1fe96713cebbaa2c10c54917R8
it is defined as `is_linked` on the serializer, which aliases to from `Defendant#linked?`

Reply:
Added the correct reference.